pkg/codegen/utils.go

```diff
@@ -612,3 +613,18 @@ func EscapePathElements(path string) string {
 	}
 	return strings.Join(elems, "/")
 }
+
+// IsMediaTypeJson checks if the media type provided falls into the json kind.
+func IsMediaTypeJson(mediaType string) bool {
+	return mediaType == "application/json" || strings.HasSuffix(mediaType, "+json")
```

Comment:
rfc: this would not cover "x-json". I suggest just iterate over array and check. Or make a map and check if you hate cycles :)

Reply:
good point, reused `contentTypesJSON` - that way we are going to have single source of json types